src/main/java/fish/payara/extras/upgrade/BaseUpgradeCommand.java

```diff
@@ -102,7 +102,6 @@ public abstract class BaseUpgradeCommand extends LocalDomainCommand {
     private static final String[] CONSTANTMOVEFOLDERS = {"common",
             "config" + File.separator + "branding",
             "config" + File.separator + "osgi.properties",
-            "h2db",
```

Comment:
Hmm, I'd be careful about removing these - what about if we need to roll back an upgrade?
Needs some testing to see what happens.

Reply:
Still need to do some testing to see what happens around this.

Reply:
I put it back and it worked after merge from your PR